### PR TITLE
Update Title, Fix Blog formatting.

### PR DESCRIPTION
**Pull request by josh-padnick**
@brikis98 Take a look.

### CODE REVIEW

**Comment by josh-padnick** [comment]
On the Title, the issue is this in Google search results:

image

File: index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Gruntwork
+title: Gruntwork | A Better Way To Do DevOps
```
Comment (brikis98) [inline index.html]: How about "DevOps as a Service"?

File: _includes/post-metadata.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <div class="h6">
   <span class="mr1 nowrap">
     <i class="fa fa-calendar"></i>
-    <time datetime="{{ include.post.date | date:"%Y-%m-%d" }}">{{ include.post.date | date:"%b %d, %Y" }}</time>
+    <time datetime="{{ include.post.date | date:"%Y-%m-%d" }}">{{ include.post.date | date:"%b %-d, %Y" }}</time>
```
Comment (brikis98) [inline _includes/post-metadata.html]: What's this fix do?

Comment (josh-padnick) [inline _includes/post-metadata.html]: It's in the git commit. Changes "Oct 05, 2016" to "Oct 5, 2015" (no more leading zeros)

Comment (brikis98) [inline _includes/post-metadata.html]: Ah, cool

File: index.html (hunk 1 above)
Comment (brikis98) [inline index.html]: If we're making changes anyway, may as well get started :)

**Comment by brikis98** [comment]
Cool, lgtm! Merge whenever you're ready.

**Pull request merged**